assets/dev/js/editor/controls/base-data.js

```diff
@@ -1,9 +1,17 @@
 var ControlBaseView = require( 'elementor-controls/base' ),
 	TagsBehavior = require( 'elementor-dynamic-tags/control-behavior' ),
 	Validator = require( 'elementor-validator/base' ),
+	NumberValidator = require( 'elementor-validator/number' ),
+	BreakpointValidator = require( 'elementor-validator/breakpoint' ),
 	ControlBaseDataView;
 
 ControlBaseDataView = ControlBaseView.extend( {
+	validatorTypes: {
+		base: Validator,
+		number: NumberValidator,
```

Comment:
Are they in use?

Reply:
Not currently but this is generic so any control can add any validator

Reply:
Done.